front/js/script.js

```diff
@@ -0,0 +1,30 @@
+//The function below displays all products img, h3 and p on the home page
+
+fetch("http://localhost:3000/api/products")
+    .then(function(res) {
+        if (res.ok) {
```

Comment:
What happen if res.ok return false?

Reply:
Added an error feedback in case response returns otherwise than "ok". Then verifies what text is returned by the response then throws it as an error message.